priv/emqx.schema

```diff
@@ -1849,7 +1883,10 @@ end}.
                           {compress, cuttlefish:conf_get(Prefix ++ ".compress", Conf, undefined)},
                           {idle_timeout, cuttlefish:conf_get(Prefix ++ ".idle_timeout", Conf, undefined)},
                           {max_frame_size, cuttlefish:conf_get(Prefix ++ ".max_frame_size", Conf, undefined)},
-                          {mqtt_piggyback, cuttlefish:conf_get(Prefix ++ ".mqtt_piggyback", Conf, undefined)} | AccOpts(Prefix)])
+                          {mqtt_piggyback, cuttlefish:conf_get(Prefix ++ ".mqtt_piggyback", Conf, undefined)},
+                          {check_origin_enable, cuttlefish:conf_get(Prefix ++ ".check_origin_enable", Conf, undefined)},
+                          {allow_origin_absence, cuttlefish:conf_get(Prefix ++ ".allow_origin_absence", Conf, undefined)},
+                          {check_origins, cuttlefish:conf_get(Prefix ++ ".check_origins", Conf, undefined)}| AccOpts(Prefix)])
```

Comment:
I suggest that the logic of this piece should be reconstructed. This is because the three new configuration items are only applicable to the ws/wss listener. But it is not needed for tcp/ssl.

Reply:
the new configs are only added to ws[s] listeners.
having some extra transformation clauses for other listeners may look redundant but it does do any harm.

Reply:
You are right, there is no problem here.